easy-ui-react/src/SearchNav/EmphasizedText.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from "react";
+import { Text } from "../Text";
+
+export type EmphasizedTextProps = {
+  /**
+   * Emphasized text content to display.
+   */
+  children: ReactNode;
+};
+
+export function EmphasizedText(props: EmphasizedTextProps) {
```

Comment:
not entirely sure how to appropriately name this one. its not a heading and wasn't sure about calling it a title either. open to suggestions though

Reply:
i know what you mean. there's not a super obvious name for it. for me personally, `SearchNav.Title` fits the bill a little better, but that's not a strong opinion.